.github/workflows/tag.yml

```diff
@@ -23,3 +23,5 @@ jobs:
         uses: goreleaser/goreleaser-action@v1
         with:
           args: release --rm-dist
+        env:
```

Comment:
:100: 

Just curious on how to re-run, or one more release is required.

Reply:
Actually it's impossible to re run this action because it  triggers on a tag event:
```
on:		on:
  push:		  push:
    tags:		    tags:
      - v*
```
You can re-run failed actions, but they will use an old code.
Only one way to avoid a new release is  run goreleaser manually

Reply:
seems like only brew release was failing, I think we can re-run manually for brew only. /cc @golangci/team @golangci/core-team